docs/blog/newsletter-november-2018.rst

```diff
@@ -0,0 +1,118 @@
+
+Another idea was to target a few issues at a time. For example, you might start by getting everyone on board with "using" rather than "utilizing," then move on to eliminating passive voice.
+
+If you're the lone writer, you might have luck with transitioning to plain language without announcing it and addressing concerns as they come up.
+
+More ideas for winning colleagues over included:
+
+- Developing internal content using plain language to demonstrate its usefulness.
+- Explaining that plain language improves translation accuracy and reduces translation costs, if that's relevant to your content.
+
+If you need help wrangling tech docs into plain language, try looking to your marketing. Marketing materials tend to use less technical language, and they might give you ideas for simpler ways to phrase things in your tech docs.
+
+No matter what, adopting and implementing plain language seems to be a work in progress: almost everyone mentioned that they are looking for opportunities to improve.
+
+---------------------------
+How to hire a documentarian
+---------------------------
+
+How to hire is a topic of perennial interest. So fortunately, there's a great new addition to the Write the Docs website: the `Guide to Hiring and Getting Hired <https://www.writethedocs.org/hiring-guide/>`_.
+
+It came from the Writing Day at the Prague conference, when a group of documentarians got together to collect the questions that they use in interviews, and the questions they like to ask when they're *being* interviewed. All of this knowledge is now written up in the guide, aimed at helping out candidates and hiring managers alike.
+
+Thanks to `Kristy James <https://github.com/kristyj>`_ for running the session and putting the guide together, and to everyone who contributed words and ideas!
+
+---------------------------
+Getting feedback from users
+---------------------------
+
+This month, there's been a lot of discussion around gathering feedback from users on your documentation site. Here's a quick summary of what we learned.
+
+For starters, it's worth remembering that your users' attention is generally on the product they're trying to use. They won't tend to think about your docs as a whole, but rather about the product experience. So be careful asking questions about their experience of the documentation.
+
+One option is a rating system on each page (a thumbs up or down, or a "Was this helpful? Yes/No"). These can tell you about trends, but many felt they rarely got actionable feedback from these systems. A thumbs-down doesn't tell you what the problem with the page was, so it's hard to work out how to fix it. It can help a bit to ask for further information if you get a thumbs down.
+
```

Comment:
Suggest removing hyphen in "thumbs down" in second line for consistency
A thumbs down doesn't tell you...